packages/ckeditor5-table/src/tablecolumnresize/converters.ts

```diff
@@ -38,9 +38,10 @@ export function upcastColgroupElement( tableUtilsPlugin: TableUtils ): ( dispatc
 		}
 
 		const columnElements = getTableColumnElements( tableColumnGroup );
-		let columnWidths = getTableColumnsWidths( tableColumnGroup );
 		const columnsCount = tableUtilsPlugin.getColumns( modelTable );
+		let columnWidths = translateColSpanAttribute( tableColumnGroup, conversionApi.writer );
 
+		// Fill the array with 'auto' values if the number of columns is higher than number of declared values.
 		columnWidths = Array.from( { length: columnsCount }, ( _, index ) => columnWidths[ index ] || 'auto' );
 
 		if ( columnWidths.length != columnElements.length || columnWidths.includes( 'auto' ) ) {
```

Comment:
This is not triggering column width normalization for such case:
```html
<table>
    <colgroup>
        <col style="width: 30pt;">
        <col style="width: 70pt;">
    </colgroup>
    <tbody>
        <tr>
            <td>a</td>
            <td>b</td>
        </tr>
    </tbody>
</table>
```
And the final table ends up like this:
```html
<figure class="table">
    <table class="ck-table-resized">
        <colgroup>
            <col style="width:30%;">
            <col style="width:70%;">
        </colgroup>
        <tbody>
        <tr>
            <td>a</td>
            <td>b</td>
        </tr>
        </tbody>
    </table>
</figure>
```
so `pt` values are treated as they would be `%`, the table gets full page width and the original sizes are not preserved.

The column width is normalized in post-fixer (but should be normalized before post-fixers are triggered).